extra/liquidexport

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""liquidexport – host a metrics HTTP endpoint with Prometheus formatted data from liquidctl
+
+This is an experimental script that collects stats from liquidctl and exposes them as a http://localhost:8098/metrics
+endpoint in the Prometheus text format.
+See: https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-example
+
+Example metric with labels:
+# HELP liquidctl liquidctl exported metrics
+# TYPE liquidctl gauge
+liquidctl{device="NZXT Kraken X (X42, X52, X62 or X72)",sensor="liquid_temperature",unit="°C"} 33.6
+
+Usage:
+  liquidexport [options]
+  liquidexport --help
+  liquidexport --version
+
+Options:
+  --interval <seconds>    Update interval in seconds [default: 2]
+  --hid <module>          Override API for USB HIDs: usb, hid or hidraw
+  --legacy-690lc          Use Asetek 690LC in legacy mode (old Krakens)
+  --match <name>          Filter devices by name
+  --vendor <id>           Filter devices by vendor id
+  --product <id>          Filter devices by product id
+  --release <number>      Filter devices by release number
+  --serial <number>       Filter devices by serial number
+  --bus <bus>             Filter devices by bus
+  --address <address>     Filter devices by address in bus
+  --usb-port <port>       Filter devices by USB port in bus
+  --pick <number>         Pick among many results for a given filter
+  --port <number>         Port for the HTTP /metrics endpoint
+  -v, --verbose           Output additional information
+  -g, --debug             Show debug information on stderr
+  --version               Display the version number
+  --help                  Show this message
+
+Examples:
+  liquidexport
+  liquidexport --product 0xb200
+  liquidexport --interval 0.5
+
+Copyright (C) 2020  Alex Berryman
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import logging
+import sys
+import time
+
+import liquidctl.cli as _borrow
+import usb
+
+from docopt import docopt
+from liquidctl.driver import *
+
+from prometheus_client import start_http_server
+from prometheus_client.core import GaugeMetricFamily, REGISTRY
+
+import re
+
+LOGGER = logging.getLogger(__name__)
+
+
+def gauge_name_sanitize(name):
+    name = re.sub(r'(?<!^)(?=[A-Z])', '_', name).lower()
```

Comment:
This regex substitution will result in weird output for something like:

```python
>>> some_status_item = ('LED accessories', 2, '')  # from Smart Device V1
>>> re.sub(r'(?<!^)(?=[A-Z])', '_', some_status_item[0]).lower()
'l_e_d accessories'
```

Admittedly one could argue that this specific metric should be filtered out, but "LED" and other acronyms are bound to appear in more interesting places.

Reply:
Very good point. 

Re-evaluating the name used again I think I can delete the regex and just replace it with two simple steps:
1. Replace spaces with `_`
2. String to lower

```python
def gauge_name_sanitize(name):
    return name.replace(" ", "_").lower()
```